src/locale/messages.fr.xlf

```diff
@@ -9,9 +9,31 @@
         <source>Oops, we are unable to make the site work properly. Are you connected to the Internet?</source><target state="translated">Impossible de démarrer une session. Êtes-vous bien connecté à Internet ?</target>
```

Comment:
@smadbe with such a task, a lot of translations will be lost :confused: 
Do we have to rework the french translation file **before** merging ?

Reply:
I will redo them, no problem.